pkg/ebpf/c/tracee.bpf.c

```diff
@@ -752,7 +752,7 @@ typedef struct event_context {
     u64 ts; // Timestamp
     task_context_t task;
     u32 eventid;
-    u32 padding; // free for further use
+    u32 syscall; // The syscall which triggered the
```

Comment:
Triggered the event

Reply:
Still missing the end there
```suggestion
    u32 syscall; // The syscall which triggered the event
```